test/tests/interfaces/pbs_partition.py

```diff
@@ -157,7 +156,9 @@ def test_set_partition_to_routing_queue(self):
         """
         Test to check the set of partition attribute on routing queue
         """
-        msg1 = "Can not assign a partition to route queue"
+        msg0 = "Route queues are incompatible with the "\
+               "partition attribute enabled"
+        msg1 = "Cannot assign a partition to route queue"
```

Comment:
msg2 will be logged in case assertTrue fails. Make the message intuitive. May be something like "Qmgr error message do not match"